xtevent/xtevent.sthlp

```diff
@@ -192,9 +192,12 @@ trend (_ttrend) to the regression. {opt (gmm)} uses the GMM to compute the trend
 {opt saveov:erlay} saves estimations for the overlay plot produced by {opt xteventplot, overlay(trend)}.
 
 {phang}
-{opt savek(stub)} saves variables for event-time dummies, event-time and trends. Event-time dummies are stored as {it: stub}_eq_m# for the dummy
+{opt savek(stub, [subopt])} saves variables for event-time dummies, event-time and trends. Event-time dummies are stored as {it: stub}_eq_m# for the dummy
 variable # periods before the policy change, and {it:stub}_p# for the dummy variable # periods after the policy change. The dummy variable for
-the policy change time is {it:stub}_p0. Event time is stored as {it:stub}_evtime. The trend is stored as {it:stub}_trend.
+the policy change time is {it:stub}_p0. Event time is stored as {it:stub}_evtime. The trend is stored as {it:stub}_trend. The following ca be passed as a suboption:
+

```

Comment:
@Constantino-Carreto-Romero typo. Also, this feels like a clunky way to document a suboption. Can you see helpfile examples to see if there's a better way?

Reply:
Change the documentation to reflect that we only have one (optional) suboption. Use `gmm` as an example.

Reply:
@jorpppp I have changed the suboption format, so now when there is only one suboption what is shown is the suboption name instead of the word `suboption`.